File: sprint03/JMS/PGS_42576.cpp
#include <string>
#include <algorithm>
#include <vector>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    
    sort(participant.begin(), participant.end());
    sort(completion.begin(), completion.end());
    
    for(int i=0;i<completion.size();i++){
        if(completion[i].compare(participant[i]) != 0){
            answer = participant[i];
            return answer;
        }
    }
    
    answer = participant[participant.size()-1];
    return answer;
}

File: sprint03/KSH/PGS_42576.py
def solution(participant, completion):
    answer = ''
    dic = dict()
    for i in participant:
        if i in dic.keys():
            dic[i] += 1
        else:
            dic[i] = 1
    
    for i in completion:
        dic[i] -= 1
    
    for i in dic:
        if dic[i] == 1:
            answer = i
            break
    
    return answer